### PR TITLE
add additional information on dealing with ~/.ssh/config errors on Mac

### PR DESCRIPTION
**Pull request by michaellihs**
Given the many upvotes on this post on StackOverflow https://stackoverflow.com/questions/47455300/ssh-config-bad-configuration-option-usekeychain-on-mac-os-sierra-10-12-6 it seems like many users have an issue with Mac not dealing properly with the `UseKeychain` config in the `~/.ssh/config`. This commit adds additional information on how to fix the problem and allows GitHub users to find a solution in the GitHub documentation when searching for the error message.

### Why:

Given the many upvotes on this post on StackOverflow https://stackoverflow.com/questions/47455300/ssh-config-bad-configuration-option-usekeychain-on-mac-os-sierra-10-12-6 it seems like many users have an issue with Mac not dealing properly with the `UseKeychain` config in the `~/.ssh/config`. This commit adds additional information on how to fix the problem and allows GitHub users to find a solution in the GitHub documentation when searching for the error message.

### What's being changed:

Add additional information for Mac users on how to fix the `~/.ssh/config` configuration on their Macs. Diff should be self-explanatory.

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
@michaellihs Thanks so much for opening a PR! I'll get this triaged for review ⚡

File: content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md [hunk 1]
```diff
@@ -105,6 +105,23 @@ Before adding a new SSH key to the ssh-agent to manage your keys, you should hav
...
+     ```
+     
+     you need to add an additional config line to your `Host *` section:
+     
+     ```
+     Host *
+       IgnoreUnknown UseKeychain
+     ```
+     
+     {% endnote %}
```
Comment (skedwards88) [inline content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md:124]:
```suggestion
      {% mac %}
      {% note %}

      **Note:** If you see an error like this

      ```
      /Users/USER/.ssh/config: line 16: Bad configuration option: usekeychain
      ```

      add an additional config line to your `Host *` section:

      ```
      Host *
        IgnoreUnknown UseKeychain
      ```

      {% endnote %}
      {% endmac %}
```

**Review by skedwards88 — APPROVED** [review]
✨ Thank you for this addition! We have a liquid tag that enables you to display OS-specific content, so I applied this to your PR.

I'll get this merged down for you.

**Comment by michaellihs** [comment]
> ✨ Thank you for this addition! We have a liquid tag that enables you to display OS-specific content, so I applied this to your PR.
> 
> I'll get this merged down for you.

Thanks for letting me know and thanks for merging :)

**Pull request merged**